virttest/utils_test/libvirt.py

```diff
@@ -1495,8 +1495,8 @@ def create_hostdev_xml(pci_id, boot_order=None,
                      'bus': device_bus, 'function': device_function}
             hostdev_xml.source = hostdev_xml.new_source(**attrs)
         if dev_type == "usb":
-            addr_bus = pci_id.split(':')[2]
-            addr_device = pci_id.split(':')[3]
+            addr_bus = pci_id.split(':')[2].lstrip("0")
```

Comment:
I suppose  it should be expected as a hexical number, right? like  `bus='0x08'`
If then, I think we should use better way to fix instead easily removing the all left '0'.
So could you try similar below?
` addr_bus = hex(int(pci_id.split(':')[2]))`

Reply:
yeah, the output of lscpi should be hexical numbers.
Thanks, Update.